scipy/special/add_newdocs.py

```diff
@@ -161,6 +161,62 @@ def add_newdoc(place, name, doc):
 
     """)
 
+
+add_newdoc("scipy.special", "agm",
+    """
+    agm(a, b)
+
+    Compute the arithmetic-geometric mean of `a` and `b`.
```

Comment:
Actually, the docstring standard says to use single backticks around the names of parameters; see subsection 4 ("Parameters") of the "Sections" section at https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt